src/Elmish.WPF/WpfProgram.fs

```diff
@@ -21,21 +21,32 @@ type WpfProgram<'model, 'msg> =
 module WpfProgram =
 
 
-  let private create getBindings program =
+  let private create createViewModel program =
     { ElmishProgram = program
-      Bindings = getBindings ()
+      CreateViewModel = createViewModel
       LoggerFactory = NullLoggerFactory.Instance
       ErrorHandler = fun _ _ -> ()
       PerformanceLogThreshold = 1 }
 
+  let createDynamic (getBindings: unit -> Binding<'model, 'msg> list) performanceLogThreshold bindingsLogger performanceLogger : ('model * ('msg -> unit) -> DynamicViewModel<'model, 'msg>) =
+    let bindings = getBindings ()
```

Comment:
I wonder if this line should be inside the body of the returned function?

Reply:
We actually ran into this issue with `Guid` generation in the bindings project. In certain cases the `Guid`'s would regenerate each time, but in other cases they would only regenerate once. Of course the fix was to run the `Guid` generation _inside_ the `cmd` binding or setter.